ext/gd/config.m4

```diff
@@ -119,27 +126,34 @@ AC_DEFUN([PHP_GD_FREETYPE2],[
 
     PHP_EVAL_INCLINE([$FREETYPE2_CFLAGS])
     PHP_EVAL_LIBLINE([$FREETYPE2_LIBS], [GD_SHARED_LIBADD])
-    AC_DEFINE(HAVE_LIBFREETYPE, 1, [ ])
-    AC_DEFINE(HAVE_GD_FREETYPE, 1, [ ])
+    AC_DEFINE([HAVE_LIBFREETYPE], [1],
+      [Define to 1 if you have the FreeType library.])
+    AC_DEFINE([HAVE_GD_FREETYPE], [1],
+      [Define to 1 if gd extension has FreeType support.])
   fi
 ])
 
 AC_DEFUN([PHP_GD_JISX0208],[
   if test "$PHP_GD_JIS_CONV" = "yes"; then
-    AC_DEFINE(USE_GD_JISX0208, 1, [ ])
-    AC_DEFINE(JISX0208, 1, [ ])
+    AC_DEFINE([USE_GD_JISX0208], [1],
+      [Define to 1 if gd extension has JIS-mapped Japanese font support.])
+    AC_DEFINE([JISX0208], [1],
+      [Define to 1 if GD library has JIS-mapped Japanese font support.])
   fi
 ])
 
+dnl
+dnl PHP_GD_CHECK_FORMAT(format, [action-if-found])
+dnl
 dnl Build and run a program to determine if GD has support for the given
 dnl format. The first argument is the proper-noun-capitalized name of the
 dnl format -- basically the word Foo in gdImageCreateFromFoo -- such as
-dnl Png. If support for format Foo exists, the second argument (the name
-dnl of a constant) will be defined to 1. The reason for this charade is
-dnl that gd defines "junk" versions of each gdImageCreateFromFoo function
-dnl even when it does not support the Foo format. Those junk functions
-dnl display a warning but eventually return normally, making a simple link
-dnl or run test insufficient.
+dnl Png. If support for format Foo exists, the "action-if-found" is executed.
+dnl The reason for this charade is that gd defines "junk" versions of each
+dnl gdImageCreateFromFoo function even when it does not support the Foo format.
+dnl Those junk functions display a warning but eventually return normally,
+dnl making a simple link or run test insufficient.
```

Comment:
Maybe instead of `"junk"` use something like `no-op`.

Reply:
Of course. Replaced with no-op.